src/Model/MethodGroupGo.cs

```diff
@@ -86,7 +87,7 @@ internal void Transform(CodeModelGo cmg)
                     {
                         mg.ReturnType.Body.AddImports(imports);
                     }
-                    if (!string.IsNullOrEmpty(mg.ParameterValidations))
+                    if (mg.ShouldValidate())
                         validationImports = true;

```

Comment:
Minor nit, you could just do this.
```
validationImports = mg.ShouldValidate()
```

Reply:
Actually no you can't, never mind.

Reply:
You could do this though!

``` C#
useValidation |= mg.ShouldValidate();
```
That said, I'm not sure this is any more readable.